src/Sulu/Bundle/AdminBundle/Resources/js/views/FormOverlayList/tests/FormOverlayList.test.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import mockReact from 'react';
+import {mount} from 'enzyme';
+import {observable} from 'mobx';
+import FormOverlayList from '../FormOverlayList';
+import List from '../../List';
+import Overlay from '../../../components/Overlay';
+import ResourceStore from '../../../stores/ResourceStore';
+import ResourceFormStore from '../../../containers/Form/stores/ResourceFormStore';
+import Form from '../../../containers/Form';
+import ErrorSnackbar from '../ErrorSnackbar';
+
+const React = mockReact;
+
+jest.mock('../../List', () => class ListMock extends mockReact.Component {
+    listStore = {};
+
+    render() {
+        return <div>list view mock</div>;
+    }
+});
+
+jest.mock('../../../containers/Form', () => class ListMock extends mockReact.Component {
+    render() {
+        return <div>form container mock</div>;
+    }
+});
+
+jest.mock('../../../utils/Translator', () => ({
+    translate: jest.fn((key) => key),
+}));
+
+jest.mock('../../../stores/ResourceStore', () => jest.fn(
+    (resourceKey, itemId) => {
+        return {
+            id: itemId,
+        };
+    }
+));
+jest.mock('../../../containers/Form/stores/ResourceFormStore', () => jest.fn(
+    (resourceStore) => {
+        return {
+            id: resourceStore.id,
+        };
+    }
+));
+
+let router;
+beforeEach(() => {
+    router = {
```

Comment:
This way you are coupling all of these tests with each other... I would prefer to have the router defined whatever is needed in each single test. We have similar setups like this in our PHP tests, and they are not very easy to maintain, because if you touch this variable a lot of tests will start to break :confused: